viz.yaml

```diff
@@ -64,6 +70,20 @@ fetch:
     scripts: [scripts/fetch/map_data.R , scripts/fetch/utils.R]
     depends: viewbox_limits
   -
+    id: precip_grid_data
+    location: cache/precip_grid_data.rds
+    reader: rds
+    fetcher: precip_grid_data
+    scripts: scripts/fetch/precip_grid_data.R
+    webdata_args:
+      times: ["2014-01-01", "2014-01-04"]
```

Comment:
Can the storm date range be in `parameter` in case it needs to be used to pull additional data or in the text?

Reply:
yes, see comment below that set of lines in the yaml: 
```
comment: "times and other metadata will be moved to another target, e.g., timesteps"
```